TwitchLib.EventSub.Webhooks.Example/Program.cs

```diff
@@ -1,20 +1,33 @@
-using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using TwitchLib.EventSub.Webhooks.Example;
+using TwitchLib.EventSub.Webhooks.Extensions;
 
-namespace TwitchLib.EventSub.Webhooks.Example
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddControllers();
```

Comment:
question: maybe we should remove everything related to controllers because they are not needed